core/src/main/java/org/elasticsearch/plugins/PluginsService.java

```diff
@@ -151,7 +164,18 @@ public PluginsService(Settings settings, Path pluginsDirectory, Collection<Class
             }
         }
 
-        logger.info("loaded {}, sites {}", jvmPlugins.keySet(), sitePlugins);
+        // we don't log jars in lib/ we really shouldnt log modules,
+        // but for now: just be transparent so we can debug any potential issues
```

Comment:
++
That's super important.

That said it could be also logged as debug on a specific line:

```java
logger.info("loaded {}, sites {}", jvmPluginNames, sitePlugins);
logger.debug("modules {}", moduleNames);
```